### ISSUE TITLE
Compiling lxqt from source - problem with display managers

### ISSUE THREAD

**Issue by rubyFeedback** [lxqt/lxqt#1663]
Hello. I compile most software from source and I tried my luck with lxqt.

I actually could manage to compile it just fine - but starting it does not work that well; or perhaps, it is a bit confusing to me ...

The URLs I was using were:

https://downloads.lxqt.org/downloads/libfm-qt/0.14.0/libfm-qt-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/liblxqt/0.14.0/liblxqt-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/libqtxdg/3.3.0/libqtxdg-3.3.0.tar.xz
https://downloads.lxqt.org/downloads/libqtxdg/3.2.0/?C=M;O=D
https://downloads.lxqt.org/downloads/libsysstat/0.4.2/libsysstat-0.4.2.tar.xz
https://downloads.lxqt.org/downloads/libsysstat/0.4.1/?C=M;O=D
https://downloads.lxqt.org/downloads/lxqt-about/0.14.0/lxqt-about-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/lxqt-admin/0.14.0/lxqt-admin-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/lxqt-build-tools/0.6.0/lxqt-build-tools-0.6.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-build-tools/
https://downloads.lxqt.org/downloads/lxqt-config/0.14.0/lxqt-config-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-config/
https://downloads.lxqt.org/downloads/lxqt-globalkeys/0.14.0/lxqt-globalkeys-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-globalkeys/
https://downloads.lxqt.org/downloads/lxqt-l10n/0.13.0/lxqt-l10n-0.13.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-globalkeys/
https://downloads.lxqt.org/downloads/lxqt-notificationd/0.14.0/lxqt-notificationd-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/lxqt-openssh-askpass/0.14.0/lxqt-openssh-askpass-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-openssh-askpass/
https://downloads.lxqt.org/downloads/lxqt-panel/0.14.0/lxqt-panel-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/lxqt-policykit/0.14.0/lxqt-policykit-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-policykit/
https://downloads.lxqt.org/downloads/lxqt-powermanagement/0.14.0/lxqt-powermanagement-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/lxqt-qtplugin/0.14.0/lxqt-qtplugin-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/liblxqt/
https://downloads.lxqt.org/downloads/lxqt-runner/0.14.0/lxqt-runner-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-runner/
https://downloads.lxqt.org/downloads/lxqt-session/0.14.0/lxqt-session-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-session/
https://downloads.lxqt.org/downloads/lxqt-sudo/0.14.0/lxqt-sudo-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-sudo/
https://downloads.lxqt.org/downloads/lxqt-themes/0.14.0/lxqt-themes-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-runner/
https://downloads.lxqt.org/downloads/obconf-qt/0.14.0/obconf-qt-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/obconf-qt/
https://downloads.lxqt.org/downloads/pavucontrol-qt/0.14.0/pavucontrol-qt-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/lxqt-runner/
https://downloads.lxqt.org/downloads/pcmanfm-qt/0.14.0/pcmanfm-qt-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/pcmanfm-qt/
https://downloads.lxqt.org/downloads/qterminal/0.14.0/qterminal-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/qterminal/
https://downloads.lxqt.org/downloads/qtermwidget/0.14.0/qtermwidget-0.14.0.tar.xz
https://downloads.lxqt.org/downloads/qterminal/

(don't mind the second entries there, they are a bit wrong; but the URLs to the .tar.xz should be correct. The above output was generated from a script which I just lazily copy/pasted into this textfield widget)

I start **lxqt** via:

exec ck-launch-session dbus-launch --exit-with-session startlxqt

in .xinitrc

I also manage to start ... something. lxqt comes up with a "choose a display
manager". This is the first thing that confuses me - do I need a display manager?
I login through .xinitrc for xfce, mate-desktop, fluxbox etc... I don't even HAVE a
display manager. This is actually what stops me altogether - that is a blockage
to me. Since I have no display manager, the popup widget stops me from
continuing to go. I tried to use xdm as display manager but that widget does
not even allow me to pick anything ... which is quite frustrating (I can navigate
through the listing of files but not pick anything).

It may be that I am missing something in my setup, but even then I think it
would be useful if lxqt could be changed a little bit to allow for more 
flexibility.

I will next give some possible entry points which could be improved here -
please pick whatever you deem useful and just ignore the rest; I have no
real insight into the lxqt project so I am quite clueless, but I am coming
here from the point of view of a medium-advanced user, give or take.

(1) Consider making that strange widget that popups optional. I am not
sure if this is possible but do we need any external display manager? 
Perhaps a new button could be added "e. g. skip" or something like that.

(2) if lxqt requires a window manager, as a mandatory part, perhaps
a minimal one could be added to the lxqt codebase? 

(3) It would be helpful if somewhere in the widget, or anywhere else,
some info box or so, a listing of supported display managers could
be shown. For example, would "kwin" work? Is that even a display
manager? I have no idea; I also can't tell from that widget either.

(4) IF a display manager is necessary, could some compilation
support be added to whatever needs it? That way I could
use a configure option to cmake and then bypass that check
altogether; additionally OR optionally, perhaps an additional
ENVIRONMENT variable could be used to specify which
display manager to use then.

(5) I would perhaps postpone this whole check; perhaps first
start lxqt anyway, and then use some notification widget
where users can click on to set a DM. That way I could
click away that widget and use lxqt. Right now I do not
even get past that point at all (it may be that something
is wrong on my system in general, but even then I think
it would be helpful if more information is provided to the user
in this event. Perhaps also have a toggle check box to 
disallow starting this notification widget at a later time,
in the event that the user does not need any display
manager. I understand that many people need this and
use this but I don't need it.

Hmm. I think I'll stop here - the most important point is
not the 5 suggestions but more the general wish to see
that I can successfully use lxqt. I have compiled all these
components but can not use lxqt right now due to the
widget popping up (I am actually quite sure that I also
did something wrong because no other panels appear,
but even in this case I think it may be helpful to show 
more context and more information, so that I can find
out what has gone wrong altogether).

As for my system, it's a linux system with gcc 8.2.0,
latest binutils git cloned source, all of xorg compiled
from latest stable sources, glibc quite old though at
2.23 and linux kernel also quite old at 4.4.14 - but
despite being old, the latter two work quite ok for my
use cases so far.

Anyway thanks for reading this! I hope the next lxqt
release can be a bit improved - I can compile mate-desktop
and xfce just fine and start them (I am writing this from
a self-compiled xfce right now for example).

PS: Sorry if I mixed up the terms window manager with
display manager; I know these are not the same but I am often
confused so please read the above accordingly.

**Comment by agaida** [comment]
TLDR; - and not filled out bug form. Really - we put it in for a reason.

**Comment by tsujan** [comment]
@rubyFeedback As human beings, we have a limited free time and want to use it as effectively as possible. Attending to such "reports" would waste everyone's time. That's why @agaida closed it immediately.

I added this comment to emphasize that even if you fill in the template, each report should be only about *one clear problem*.
